app/Http/Controllers/Api/v1/GameController.php

```diff
@@ -18,13 +18,13 @@ public function index(){
             ->get();
     }
 
-    public function show($id){
+    public function show($id) {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->